server/api/service/geocoder.js

```diff
@@ -23,35 +24,75 @@ const degreeToMeter = (lon, lat, dn, de) => {
 // Some parcels (Gushim) are malformed (usually in the Golan Heights) and should be ignored.
 const BAD_PARCELS = ['203000', '202011', '202000', '201000', '200000', '200001'];
 
-const gushHelkaToPolygon = async (gush, helka) => {
+const gushHelkaToPolygon = async (gush, helka) => {		
 	if (!gush || !helka) {
 		return;
 	}
+	
 	if (BAD_PARCELS[gush]) {
 		return;
 	}
+
+
+	// parse the raw string into a unique array of numbers 
+	// e.g -
+    //  - "123,123" => [123]
+	//  - "1-2" => [1,2]
+	//  - "helka 4 and 5" => [4,5]  
+	const helkaArr = [...new Set(helka.match(/\d+/gi))]
```

Comment:
consider adding range:
"1-3" => [1,3] => [1,2,3]

Reply:
will do when we'll have a DB for this data